mpd/fixtures/hbbtv_profile.mpd

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" profiles="urn:hbbtv:dash:profile:isoff-live:2012" type="static" mediaPresentationDuration="PT6M16S" minBufferTime="PT1.97S">
```

Comment:
For HBBTV 1.5, I believe you should define both the HBBTV profile, and the Live Profile, IE: `urn:hbbtv:dash:profile:isoff-live:2012,urn:mpeg:dash:profile:isoff-live:2011`